server/ansible/roles/pbench-server-vars/tasks/main.yml

```diff
@@ -0,0 +1,162 @@
+---
+# Get variables from the config file on the server and register them
+# as facts, so they can be used in all subsequent roles. It is assumed
+# that the config file has already been installed on the server.
+
+# pbench top level directory
+- name: fetch the top-level pbench directory from remote
+  shell: getconf.py pbench-top-dir pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_dir }}"
+    verbosity: 1
+
+# pbench logs dir
+- name: fetch the logs directory
+  shell: getconf.py pbench-logs-dir pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_logs_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_logs_dir }}"
+    verbosity: 1
+
+# pbench tmp dir
+- name: fetch the tmp directory
+  shell: getconf.py pbench-tmp-dir pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_tmp_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_tmp_dir }}"
+    verbosity: 1
+
+# pbench quarantine dir
+- name: fetch the quarantine directory
+  shell: getconf.py pbench-quarantine-dir pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_quarantine_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_quarantine_dir }}"
+    verbosity: 1
+
+# pbench hostinfo prefix
+- name: fetch the host info prefix
+  shell: getconf.py host-info-path-prefix results
+  register: cmd_output
+  ignore_errors: yes
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_host_info_prefix: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_host_info_prefix }}"
+    verbosity: 1
+
+# pbench reception area
+- name: fetch the reception area directory
+  shell: getconf.py pbench-receive-dir-prefix pbench-server
+  register: cmd_output
+  ignore_errors: yes
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_reception_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_reception_dir }}"
+    verbosity: 1
+
+# httpd document root
+- name: fetch the httpd document root directory
+  shell: getconf.py documentroot apache
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
```

Comment:
Is the `/opt/pbench-server` configurable?

Reply:
See above.